app/models/vim_performance_operating_range.rb

```diff
@@ -6,22 +6,35 @@ class VimPerformanceOperatingRange < ApplicationRecord
 
   DEFAULT_STD_DEV_MULT = 1
 
+  def recalculate_values
+    self.values = Metric::LongTermAverages.get_averages_over_time_period(
+      resource,
+      :avg_days    => days,
+      :ext_options => {:time_profile => time_profile}
+    )
+  end
+
   def values_to_metrics(options = {})
-    options[:std_dev_mult] ||= DEFAULT_STD_DEV_MULT
+    @values_with_metrics ||= begin
+      results = values_with_std_dev(options[:std_dev_mult])
+      Metric::LongTermAverages::AVG_METHODS_INFO.each_with_object({}) do |(meth, info), h|
+        h[meth.to_s] = results.fetch_path(info[:type], info[:column])
+      end
```

Comment:
I named values_with_std_dev as such because it's the exact same format as values but with extra stuff.  This part converts "values" format to "metrics" format